README.md

```diff
@@ -145,6 +147,20 @@ psql postgres://postgres:password@localhost:5432/postgres
 
 You can also find a SQL runner in Hasura itself, [here](http://localhost:9695/data/sql).
 
+# Localization
+
+Translation management is powered by our friends at [locize](https://locize.com/).
+
+## Development
+
+English fallback strings, inlined throughout the code represent the source of truth. When [configured](.env.local.sample) with a [locize API key](https://www.locize.app/p/9svqlsjm/developer), new keys and updated default values will by synchronized to locize during local development.
+
+## Production
+
+Non-English language resources are served via a cached (1 day) [`production` version](https://www.locize.app/p/9svqlsjm/v/production) on [locize's CDN](https://docs.locize.com/guides-tips-and-tricks/going-production#use-the-locize-cdn).
+
+Given the inlined fallback strings, it is unnecessary to load external English resources; an empty in-memory `en` language is provided to i18next via [`partialBundledLanguages`](https://www.i18next.com/overview/configuration-options#languages-namespaces-resources).
+
```

Comment:
We should at least add some documentation that describe how one would go about contributing to translations, but here's ☝🏻 a brief summary of how I'm thinking about the process in the meantime.